tests/test_other.py

```diff
@@ -8684,8 +8684,8 @@ def test_js_optimizer_parse_error(self):
 var ASM_CONSTS = [function() { var x = !<->5.; }];
                                         ^
 ''', '''
-var ASM_CONSTS = [function() {var x = !<->5.;}];
-                                       ^
+  0: function() {var x = !<->5.;}
```

Comment:
Nit: This condition should still check the shape of the whole dictionary? At least including the ASM_CONSTS part to make sure it's checking what we think it is (I am not sure how this test works, mind)